docs/Tutorials/Quickstarts/Private-Network-Quickstart.md

```diff
@@ -320,112 +325,88 @@ npm install -g truffle
 
 !!! note
     `npm` requires `sudo` on Linux.
+    
 
-Create a `pet-shop-tutorial` directory and move into it:
-
-```bash
-mkdir pet-shop-tutorial
-
-cd pet-shop-tutorial
-```
-
-Unpack Pet Shop [Truffle box](https://truffleframework.com/boxes): 
-
-`truffle unbox pet-shop`
+The source code for the smart contract and DApp are in the `pet-shop` folder. With the blockchain running and Metamask connected to it, run the command:
 
-Install the [Truffle wallet](https://github.com/trufflesuite/truffle-hdwallet-provider):
-Note with Truffle 5, you must use a Web3 1.0 enabled wallet or the Truffle tasks will hang.
+ ```bash tab="Linux/MacOS"
+ ./run-dapp.sh
+ ```
 
-```bash
-npm install --save @truffle/hdwallet-provider
-```
+The script first compiles the contract, then performs a migration to the blockchain network and then runs some tests. 
+You can then search for the deployed contracts and transactions in the block explorer using the addresses displayed in your output.
+Output similar to the following is displayed (your addresses will differ):
 
-!!!note
-    `npm` requires `sudo` on Linux.
+```log
+Compiling your contracts...
+===========================
+> Everything is up to date, there is nothing to compile.
```

Comment:
it should not output that it's already compiled if it's the first time you compile it.

Reply:
:+1: fixed